packages/server/src/resolvers/Order/OrderResolver.ts

```diff
@@ -61,6 +61,13 @@ class OrderResolver {
     return await OrderModel.find({ store: store._id }).exec();
   }
 
+  @Query((_type) => Order)
+  async order(
+    @Arg('id', (_type) => ObjectIdScalar) id: ObjectId,
+  ): Promise<Order | null> {
+    return await OrderModel.findById(id).populate('items.productOption.image');
```

Comment:
you should be able to use
```suggestion
    return await OrderModel.getFromUserInputId(id).populate('items.productOption.image');
```

and it will handle throwing with a nice error if the id does not correspond to an existing order

Reply:
or you may have to call the populate like this
```ts
const order = await OrderModel.getFromUserInputId(id);
await order.populate('items.productOption.image');
```